specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Subscriptions.json

```diff
@@ -0,0 +1,80 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-05-02-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/getVerificationId": {
+      "post": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "description": "Get the verification id of a subscription used for verifying custom domains",
+        "operationId": "GetVerificationId",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
@Seris370  I want to have a better understanding of this scenario, Is this operation supposed to work on multiple tenants or subscriptions ? you expect the subscriptionId in this operation as a client level parameter ? 
I mean if it's a client level parameter, customer will have user experience like 
```
const client = new AppContainerManagementClient(credential, subsId1);
client.getVerificationId()
```
if customer wants to query another subscriptionId, they will need to create another client 
```
const client2 = new AppContainerManagementClient(credential, subsId2);
client2.getVerificationId()
```
If it's a tenant level, we expect the user experience like 
```
const client = new AppContainerManagementClient(credential);
client.getVerificationId(subsId1);
client.getVerificationId(subsId2);
```
we should make this subscriptionId as a method level, where you should not import from the common type. Instead you should have your own subscriptionId parameter and set x-ms-parameter-location as method.